sdk/KoreBuild/KoreBuild.Common.props

```diff
@@ -37,6 +37,7 @@ Default layout and configuration.
     <ArtifactsDir>$(RepositoryRoot)artifacts\</ArtifactsDir>
     <BuildDir>$(ArtifactsDir)build\</BuildDir>
     <IntermediateDir>$(RepositoryRoot)obj\</IntermediateDir>
+    <ProjectType>Solution</ProjectType>
```

Comment:
Just an idea. Another way to do this is to follow the MSBuild convention of \*.\*proj extension being used to determine project type.

e.g., Universe might instead look like this:
```
- src/
    - Microsoft.AspNetCore.All/
        + Microsoft.AspNetCore.All.fxproj
```
```xml
<!-- Microsoft.AspNetCore.All.fxproj -->
<Project>
  <Import Project="KoreBuild.Common.props" />

  <ItemGroup>
     <Repository Id="Antiforgery" Branch="rel/2.0.2" />
     <!-- the rest of the stuff. -->
  </ItemGroup>

  <Import Project="KoreBuild.Common.targets" />
</Project>
```

Example of this kind of layout: https://github.com/aspnet/Coherence-Final/blob/dev/packaging/1.0.6/AspNetCore.PackageCache.1.0.6.pkgproj